lib/web_ui/test/canvaskit/filter_test.dart

```diff
@@ -0,0 +1,89 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @dart = 2.6
+import 'package:test/bootstrap/browser.dart';
+import 'package:test/test.dart';
+import 'package:ui/src/engine.dart';
+import 'package:ui/ui.dart' as ui;
+
+import 'common.dart';
+
+void main() {
+  internalBootstrapBrowserTest(() => testMain);
+}
+
+void testMain() {
+  List<CkColorFilter> createColorFilters() {
+     return <CkColorFilter>[
+       EngineColorFilter.mode(ui.Color(0x12345678), ui.BlendMode.srcOver) as CkColorFilter,
+       EngineColorFilter.mode(ui.Color(0x12345678), ui.BlendMode.dstOver) as CkColorFilter,
+       EngineColorFilter.mode(ui.Color(0x87654321), ui.BlendMode.dstOver) as CkColorFilter,
+       EngineColorFilter.matrix(<double>[
+           1, 0, 0, 0, 0,
+           0, 1, 0, 0, 0,
+           0, 0, 1, 0, 0,
+           0, 0, 0, 1, 0,
+       ]) as CkColorFilter,
```

Comment:
Let's also test that we can take `Float32List` (and implement it more efficiently by skipping the conversion)

Reply:
Added a Float32List matrix. But the skipping conversion part isn't easy to test, as the matrix gets sent to an opaque skia object.

Reply:
Yeah, skipped conversion is an implementation detail. Ok not to test.